test/smokes/browserFSTransfer/CodeFolding/Prefs.js

```diff
@@ -0,0 +1,136 @@
+/**
```

Comment:
All these are assets for test files. 
Not executed as code, but used by other tests as sample test js extension